static/index.html

```diff
@@ -223,11 +223,29 @@ <h3>Last error</h3>
             v-model="organization"
             class="form-control"
             id="organization"
+            data-cy="organizationId"
           />
+          <div class="form-check mt-3">
+            <input
+              type="checkbox"
+              class="form-check-input"
+              id="org-login-check"
+              v-model="useOrgAtLogin"
+              data-cy="useOrgAtLogin"
+            />
+            <label for="org-login-check" class="form-check-label"
+              >Apply only at loginWithRedirect</label
+            >
```

Comment:
This allows us to toggle whether the organization ID is given to the constructor options, or only to `loginWithRedirect` for easy testing.